oauth2client/service_account.py

```diff
@@ -81,6 +81,10 @@ class ServiceAccountCredentials(AssertionCredentials):
                    service account.
         user_agent: string, (Optional) User agent to use when sending
                     request.
+        token_uri: string, URI for token endpoint. For convenience defaults
+                   to Google's endpoints but any OAuth 2.0 provider can be
+                   used.
+        revoke_uri: string, URI for revoke endpoint.
```

Comment:
Here and below, this is copy/paste for consistency from the parent class [AssertionCredentials](https://github.com/google/oauth2client/blob/e54e28939cf985e4d25ef34f96162abd3ab65002/oauth2client/client.py#L1594)